src/Workspaces/Core/Portable/Utilities/Documentation/XmlDocumentationProvider.cs

```diff
@@ -23,6 +23,12 @@ public static XmlDocumentationProvider Create(byte[] xmlDocCommentBytes)
             return new ContentBasedXmlDocumentationProvider(xmlDocCommentBytes);
         }
 
+        [System.Diagnostics.CodeAnalysis.SuppressMessage("Microsoft.FxCop.Rules.Security.Xml.SecurityXmlRules", "CA3053:UseXmlSecureResolver",
```

Comment:
Yes that's true (although I also like having the suppression close to the source so that someone looking at the source and wondering why we don't use the secure resolver can tell right away). I will leave it as is for now. If we run into new violations of this rule in the future for the same assemblies then I will clean this up.